packages/admin/cms-admin/src/dam/FileForm/Duplicates.tsx

```diff
@@ -98,6 +98,15 @@ const Duplicates: React.FC<{ fileId: string }> = ({ fileId }) => {
                     </ListItem>
                 )}
             </List>
+            <Alert
+                title={<FormattedMessage id="comet.dam.file.duplicates.info.title" defaultMessage="What are duplicate assets?" />}
+                sx={{ marginTop: 2 }}
```

Comment:
Please stick to the screen design. There the margin is 20px, here it's only 10

```suggestion
                sx={{ marginTop: 4 }}
```